editor/src/components/canvas/canvas-strategies/strategies/grid-resize-element-strategy.ts

```diff
@@ -104,15 +111,60 @@ export const gridResizeElementStrategy: CanvasStrategyFactory = (
         null,
       )
 
-      const gridProps = getNewGridPropsFromResizeBox(resizeBoundingBox, allCellBounds)
+      const gridPropsNumeric = getNewGridPropsFromResizeBox(resizeBoundingBox, allCellBounds)
 
-      if (gridProps == null) {
+      if (gridPropsNumeric == null) {
         return emptyStrategyApplicationResult
       }
 
       const gridTemplate =
         selectedElementMetadata.specialSizeMeasurements.parentContainerGridProperties
 
+      const elementGridPropertiesFromProps =
+        selectedElementMetadata.specialSizeMeasurements.elementGridPropertiesFromProps
+
+      const width =
```

Comment:
I don't like the term `width`/`height` because I'm always mistaken that it is a geometrical width/height (in pixels). Is there a better name for this? Maybe `columnCount` and `rowCount`

Reply:
makes total sense, fixed in [`6be5569` (#6685)](https://github.com/concrete-utopia/utopia/pull/6685/commits/6be5569b638bf408abf840f96f655e53a785ddd4)